src/main/java/com/teamacronymcoders/epos/locks/FuzzyLockKeyTypes.java

```diff
@@ -0,0 +1,14 @@
+package com.teamacronymcoders.epos.locks;
+
+import com.teamacronymcoders.epos.api.locks.IGenericFuzzyLockType;
+
+public enum FuzzyLockKeyTypes implements IGenericFuzzyLockType {
+    ARMOR,
+    ARMOR_TOUGHNESS,
+    ATTACK_DAMAGE,
+    BLOCK_HARVEST,
+    GENERIC_NBT,
+    HUNGER,
+    SATURATION,
+    TOOL_HARVEST
+}
```

Comment:
We might want this to extend "ExtendableEnum"?
Or something, unless where this is used you're checking for IGenericFuzzyLockType instead of FuzzyLockKeyTypes?

Reply:
It checks and uses `IGenericFuzzyLockType`, it basically is used for being able to create generic lock keys for fuzzy keys that don't have a "non fuzzy' definition. (See javadocs for `IFuzzyLockKey#getNotFuzzy()`). Strictly speaking this class isn't really needed, see how I use `IGenericFuzzyLockType` in `IntegerLockKey`, Though I think having it as an enum makes it a little easier to read (and implement any methods if we decide `IGenericFuzzyLockType` needs to provide any information)